### PR TITLE
Clean up some compiler warnings.

### PR DESCRIPTION
**Pull request by GrayTShirt**

### CODE REVIEW

File: src/base.c [hunk 1]
```diff
@@ -377,10 +377,11 @@ int cw_authenticate(const char *service, const char *username, const char *passw
 	};
 
 	rc = pam_start(service, creds.username, &convo, &pam);
-	if (rc == PAM_SUCCESS)
+	if (rc == PAM_SUCCESS) {
 		rc = pam_authenticate(pam, PAM_DISALLOW_NULL_AUTHTOK);
 		if (rc == PAM_SUCCESS)
 			rc = pam_acct_mgmt(pam, PAM_DISALLOW_NULL_AUTHTOK);
+	}
```
Comment (jhunt) [inline src/base.c]: Are we 100% sure about this?  I get that the whitespace implies that the braces should be there, but it _does_ change the semantics to add them.

Comment (GrayTShirt) [inline src/base.c]: I think we want to bail on account management if we fail to start, or fail to auth. In essence, if we fail on the previous pam step, then we probably don't want to try the next.

Comment (jhunt) [inline src/base.c]: I think i'd rather see this as the following (possible white-spaced cleaned-up) steps:

```
rc = pam_start(...);
if (rc == PAM_SUCCESS)
  rc = pam_authenticate(...);
if (rc == PAM_SUCCESS)
  rc = pam_acct_mgmt(...);
```

yeah, it's double the rc check, but i think it reads better.

**Comment by jhunt** [comment]
Thanks!

**Pull request merged**